lib/convection/control/stack.rb

```diff
@@ -105,7 +105,7 @@ def []=(key, value)
       end
 
       def get(*args)
-        @attributes.get(*args)
+        @attributes.get(*args, "** Warning: Could not resolve value [#{args[1]}] for stack [#{args[0]}] **")
```

Comment:
I think we could track something like stack.error_count - we'd bump this integer each time there's a null reference to a stack (or key).  We would print something sensible so that 'convection print' completes with some useful JSON for testing/debugging, but the operation would throw an exception (because stack.error_count != 0) and the template would NOT be passed to CF.

what do you think?